app/static/js/initialize_map.js

```diff
@@ -0,0 +1,119 @@
+geocoder = new google.maps.Geocoder();
```

Comment:
Can we rename this file to something like `map.js` since it includes more than initializing the map